policy/release/tasks.rego

```diff
@@ -339,3 +411,5 @@ _format_missing(o, opt) := desc if {
 	opt
 	msg := sprintf("Task %q", [o])
 } else := sprintf("Required task %q", [o])
+
+_expires_on_annotation := "build.appstudio.redhat.com/expires-on"
```

Comment:
We should probably include an explanation in the description of the policy rule(s) about how EC determines a Task is expired/deprecated.

Reply:
Something like this maybe as a comment:
```
// If the annotation is present at all, then we consider the task deprecated
// If the value is a date in the past, then we consider the task disallowed
```

But I wonder how a task author would find out about it. Do we have any docs aimed at task authors?

Reply:
I don't see any use of setting the annotation without specifying the date.

Since the value must always be set for an annotation, I don't want to allow setting different kinds of values, e.g. support `build.appstudio.redhat.com/expires-on: true` (boolean) or `build.appstudio.redhat.com/expires-on: ""` (string), only `build.appstudio.redhat.com/expires-on: 2023-04-05` (date).